jgt_tools/utils.py

```diff
@@ -113,3 +113,17 @@ def owner_name_from(origin_url):
     owner_name = owner_name.rsplit(".", 1)[0]  # Remove `.git`
     # Keep only the last two parts that remain, which are the org/owner and repo name
     return "/".join(owner_name.split("/")[-2:])
+
+
+def default_commands(command):
```

Comment:
NB: How about `has_default_commands` to make the predicate nature of the function more apparent?

Reply:
Yeah, I started with "is" but that felt too Ruby-ish, so `has_` makes a lot of sense.